src/containers/Help.js

```diff
@@ -6,7 +6,8 @@ import MarkdownComponent from '../components/Markdown'
 class Help extends React.Component {
   render() {
     let file = null
-    try { file = require(`../../gamedata/descriptions/pages/help.md`) } catch(e){}
+    let language = window.lang || 'en'
```

Comment:
Note that if window.lang is detected as 'es' and the md file doesn't exist, this will not fallback to english but error.

Reply:
I'll fix this.